cub/cub/device/device_reduce.cuh

```diff
@@ -304,6 +307,8 @@ struct DeviceReduce
       NumItemsT num_items,
       cudaStream_t stream = 0)
   {
+    CUB_BEGIN_NVTX_RANGE_SCOPE;
```

Comment:
I am unsure about the term `BEGIN` here. The beginning of the NVTX range is indeed here since the scope guard doesn't need to be the first statement in a function, so could start later than the beginning of the scope. However, the term `BEGIN` could suggest a corresponding `END` macro, which does not exist.

Reply:
I would vote for shortening to just
```suggestion
    CUB_NVTX_RANGE_SCOPE;
```

Reply:
Sounds good!